samples/nrf_rpc/protocols_serialization/server/README.rst

```diff
@@ -0,0 +1,273 @@
+.. _nrf_rpc_protocols_serialization_server:
+
+Protocols serialization: server
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Protocols serialization server runs full Bluetooth LE and OpenThread stacks and it exposes selected functions from these stacks over the UART.
+
+Requirements
+************
+
+The sample supports the following development kits for testing the network status:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+Protocols serialization server sample implements the server part of the OpenThread and Bluetooth LE API calls serialization between two devices
+that communicate with each other using the UART interface. The sample uses nRF RPC library and CBOR encoding to serialize function calls.
+
+Building and running
+********************
+
+Complete the following steps to build and flash the server sample into another nRF52840 DK:
+
+#. Open a terminal and navigate to the nRF Connect SDK installation directory.
+
+#. Navigate to the server samples's directory:
+
+   .. code-block:: console
+
+      ~/ncs/nrf$ cd samples/nrf_rpc/protocols_serialization/server
+
+
+#. Build the server sample:
+
+   .. code-block:: console
+
+      ~/ncs/nrf/samples/nrf_rpc/protocols_serialization/server$ west build -b nrf52840dk/nrf52840 -S "openthread;ble;debug"
+
+
+   Note that you can modify the list of enabled features, which by default includes **ble** support as well as **debug** logs.
+
+#. Flash the server sample into an nRF52840 DK:
+
+   .. code-block:: console
+
+      ~/ncs/nrf/samples/nrf_rpc/protocols_serialization/server$ west flash --erase
+
+Testing
+*******
+
+For testing the server sample, use the protocol serialization client sample :ref:`nrf_rpc_protocols_serialization_client`.
+
+.. _protocols_serialization_server_app_connection:
+
+Connecting client and server
+============================
+
+The client and server devices are connected using the UART peripherals.
+
+.. tabs::
+
+    .. group-tab:: nRF52840
+
+        The nRF52840 system-on-chip includes two UART peripherals: `uart0` and `uart1`. In the protocols serialization sample, the `uart0` peripheral is used for shell and logging purposes, similarly to other applications and samples, while the `uart1` peripheral is used for sending OpenThread and Bluetooth LE remote procedure calls (RPCs).
+
+        The `uart1` peripheral is configured to use the following pins:
+
+        - **P1.1**: RX
+        - **P1.2**: TX
+        - **P1.3**: RTS (hardware flow control)
+        - **P1.4**: CTS (hardware flow control)
+
+        Therefore, in order to enable the communication between the client and the server devices, connect the two nRF 52840 DKs using jumper wires according to the following diagram:
+
+        .. figure:: /images/ps_nrf52_connections.png
+            :alt: nRF52 connections
```

Comment:
Kind of disagree. Picture provides better visibility how to connect. As a customer, I do not need to count the number of pins and read what is written on the board. It is sufficient just connect to each other shown pins. 
Since I came to the project after it was drawn, and I was the first customer of this documentation, I would say that the pictures are much convenient.

Reply:
I would be worried that a customer that looks at the illustration and does not do the counting and checking names might end up connecting the pins wrong, and then blame Nordic for inaccurate instructions.

But I think we can compromise here. If we replace the bullet lists of pins with the table and mention that **GND** need to be connected, and then introduce the illustrations with "The following figure illustrates how these pins are connected:", it makes it clear that the illustration is a secondary source.

Even then, I'd then like to have the illustrations cleaned up so the connecting lines are clean and use colors from Nordic color system. I'll try to produce versions either today or tomorrow.